viz_scripts/generic_timeseries.ipynb

```diff
@@ -104,8 +107,9 @@
    "outputs": [],
    "source": [
     "# Get timestamp from known year/month/day aggregated to days\n",
-    "sel_cols_no_label_dep = ['user_id','start_local_dt_year','start_local_dt_month','start_local_dt_day','distance_miles']\n",
-    "sel_cols_with_label_dep = sel_cols_no_label_dep + ['Mode_confirm','Mode_confirm_EI(kWH)','Mode_confirm_lb_CO2']\n",
+    "sel_cols_no_label_dep = ['user_id','start_local_dt_year','start_local_dt_month','start_local_dt_day','distance']\n",
```

Comment:
hm, I see that you do pass in distance here. But if dynamic labels is false you pass in the `distance_miles` you do post-convert lb to kg but that doesn't also make the denominator go from miles to km.

```
    if (len(dynamic_labels) > 0):
        expanded_ct = compute_CO2_footprint_kg(expanded_ct, dynamic_labels)
    else:
        expanded_ct = CO2_footprint_lb(expanded_ct, 'distance_miles', 'Mode_confirm')
```

Reply:
Thanks for catching this part. Yes, now I have computed the CO2 emission to be kg/km and lb/mile respectively.
CO2_footprint_lb() has been renamed to CO2_footprint_default() which computes with default mapping from `auxiliary_file\energy_intensity.csv`. This will have initial computation of CO2 emission in `lb/mile` which is later converted into `kg/km`.